customer-wallet/src/store/modules/data/checkout.js

```diff
@@ -36,8 +36,8 @@ export function getCheckout () {
       const vendorId = process.env.REACT_APP_VENDOR_ID || 'Qmtest'
       const url = process.env.REACT_APP_MARKETPLACE_URL || 'http://localhost:4000'
       const popr = await requestPopr(url, vendorId, {
-        amount: checkoutData.price * 100,
-        currency_symbol: 'USD cents'
+        amount: checkoutData.price * 10000,
+        currency_symbol: 'satoshi'
```

Comment:
We should use tBTC as the currency symbol. The thing is that we need to use the currency_symbol to figure out which blockchain to look up the transaction. I really should have called this field "blockchain", but for now lets not change the field name, just the value set there.